### PR TITLE
Generate values based on the original content

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Review by cjeanner — COMMENTED** [review]
With that change, we MAY end with an edpm-values.yaml being generic enough to be common with, at least, 2 VA?

File: roles/ci_gen_kustomize_values/templates/hci/edpm-values/values.yaml.j2 [hunk 1]
```diff
@@ -24,20 +24,28 @@ data:
...
 {% endif %}
     nodes:
-{% for compute in cifmw_networking_env_definition.instances.keys() %}
-{%   if compute.startswith('compute') %}
-      edpm-{{ compute }}:
+{% set instances_names = []                                                                             %}
+{% set _original_nodeset = (original_content.data | default({})).nodeset | default({})                  %}
+{% set _original_nodes = _original_nodeset.nodes | default({})                                          %}
+{% set _has_computes = _original_nodes.keys() | select('contains', 'compute') | length > 0                %}
+{% set _has_cephs = _original_nodes.keys() | select('contains', 'ceph') | length > 0                      %}
```
Comment (psathyan) [inline roles/ci_gen_kustomize_values/templates/hci/edpm-values/values.yaml.j2:31]: IMHO, we could do 
```
{% set _vm_type = _original_nodes.keys() | first | split('-')[1] %}
```

Alternatively, would it be possible to add `networker` type also.

File: roles/ci_gen_kustomize_values/templates/hci/edpm-values/values.yaml.j2 [hunk 2]
```diff
@@ -24,20 +24,28 @@ data:
...
+{% for _inst in cifmw_networking_env_definition.instances.keys()                                        %}
+{%   if _inst.startswith('compute') and _has_computes or  _inst.startswith('ceph') and _has_cephs       %}
```
Comment (psathyan) [inline roles/ci_gen_kustomize_values/templates/hci/edpm-values/values.yaml.j2:33]: if we go with `_vm_type` then we can do 
```
{% if _inst.startswith(_vm_type) %}
```

**Review by rebtoor — APPROVED** [review]

**Pull request merged**